test/compress/export.js

```diff
@@ -0,0 +1,69 @@
+issue_2038_1: {
+    options = {
+        toplevel: true,
+        unused: true,
+    }
+    mangle = {
+        toplevel: true,
+    }
+    input: {
+        export var V = 1;
+        export let L = 2;
+        export const C = 3;
+    }
+    expect: {
+        export var V = 1;
+        export let L = 2;
+        export const C = 3;
+    }
+}
+
+issue_2038_2: {
+    options = {
+        toplevel: true,
+        unused: true,
+    }
+    mangle = {
+        toplevel: true,
+    }
+    input: {
+        let LET = 1;
+        const CONST = 2;
+        var VAR = 3;
+        export { LET, CONST, VAR };
+    }
+    expect: {
+        let a = 1;
+        const c = 2;
+        var n = 3;
+        export { LET as a, CONST as c, VAR as n };
+    }
+}
+
+issue_2124: {
+    options = {
+        unused: true,
+    }
+    input: {
+        {
+            export var V = 1;
+        }
```

Comment:
While I'd prefer an outright SyntaxError for these, could you please put a comment above this test saying that these non-top-level exports are invalid and are kept due to the need to preserve erroneous code? (garbage-in, garbage-out).

Is there enough information in the parser to at least emit a warning in this case, which the test could check for with `expect_warnings`?